### PR TITLE
basic mlb stuff untested

### PR DESCRIPTION
**Pull request by kelseyedmodo**
need blake to add mlb teams into mongo so I can test this

### CODE REVIEW

**Comment by blakeparkinson** [comment]
TypeError: Cannot read property 'length' of undefined
    at /Users/blake/Projects/Blake/sports/models/players.js:151:31
    at Parser.<anonymous> (/Users/blake/Projects/Blake/sports/node_modules/xml2js/lib/xml2js.js:384:20)
    at Parser.emit (events.js:95:17)
    at Object.onclosetag (/Users/blake/Projects/Blake/sports/node_modules/xml2js/lib/xml2js.js:348:26)
    at emit (/Users/blake/Projects/Blake/sports/node_modules/xml2js/node_modules/sax/lib/sax.js:615:33)
    at emitNode (/Users/blake/Projects/Blake/sports/node_modules/xml2js/node_modules/sax/lib/sax.js:620:3)
    at closeTag (/Users/blake/Projects/Blake/sports/node_modules/xml2js/node_modules/sax/lib/sax.js:861:5)
    at Object.write (/Users/blake/Projects/Blake/sports/node_modules/xml2js/node_modules/sax/lib/sax.js:1294:29)
    at Parser.exports.Parser.Parser.parseString (/Users/blake/Projects/Blake/sports/node_modules/xml2js/lib/xml2js.js:403:31)
    at Parser.parseString (/Users/blake/Projects/Blake/sports/node_modules/xml2js/lib/xml2js.js:6:61)
    at exports.parseString (/Users/blake/Projects/Blake/sports/node_modules/xml2js/lib/xml2js.js:433:19)

**Comment by blakeparkinson** [comment]
:-1: :angry:

File: models/players.js [hunk 1]
```diff
@@ -135,6 +144,19 @@ formatEUSoccerPlayers = function(response){
   return players;
 }
 
+formatMLBPlayers = function(response, team_id){
+  parseString(response, function (err, result) {
+    var str = result[Object.keys(result)[0]];
+        for (j=0; j < str.team.length; j++){ 
+          if (str.team[j].$.id == team_id){
```
Comment (blakeparkinson) [inline models/players.js]: is this just to be extra cautious? If so, maybe add an else clause outputting a message?

Comment (kelseyedmodo) [inline models/players.js]: No, this URL is actually set up with all teams listed.
[roster
  [teams
    <team Id 1 players <1> <2>..>
    <team Id 2 players <1> <2>..>
    <team Id 3 players <1> <2>..>
  ]
]

Kelsey 
Edmodo Customer Success

Comment (blakeparkinson) [inline models/players.js]: Oh. I see. It doesnt take a team id?

Comment (blakeparkinson) [inline models/players.js]: can you do ```$.trim(str.team[j].$.id) == $.trim(team_id)```  just to be extra cautious

Comment (blakeparkinson) [inline models/players.js]: then merge. looks good

Comment (kelseyedmodo) [inline models/players.js]: hmm. JQuery is still really shaky for me. 

It's not recognizing jquery on this page. getting 
ReferenceError: $ is not defined

How do I get the jquery on this page? only thing I've seen is put html script tags on an html page..
<script src="https://code.jquery.com/jquery-1.10.2.js"></script>

**Comment by blakeparkinson** [comment]
Was a bad idea. I dont know whats going on with the jquery node module. Switched to trim().

**Pull request merged**